### ISSUE TITLE
Possibility to specify not to validate SSL certificates

### ISSUE THREAD

**Issue by KGausel** [humio/python-humio#12]
**Describe the solution you'd like**
A way to specify the Humio clients to not validate SSL certificates. This is very useful for local development.

**Comment by thehilll** [comment]
I ran into the same issue, but the function calls appear to pass `**kwargs` all the way through to the ultimate call to requests, so if you want to just ignore the certificate you could use:

```
client.ingest_json_data(
    structured_data,
    verify=False
)
```
This will insert the data into humio while throwing a warning.

If you have a root CA you want to trust you could do:

```
client.ingest_json_data(
    structured_data,
    verify='/path/to/root_cert.pem'
)
```
Which inserts the data with no warning.

**Comment by AlexanderBrandborg** [comment]
@thehilll yeah, most of the library is made so you can just pass **kwargs to the actual call.
However I'll see if I can make this a bit easier in the future so that won't be necessary. Will strive to add this soon 👍🏻
